tests/chainerx_tests/unit_tests/routines_tests/test_activation.py

```diff
@@ -72,6 +78,49 @@ def forward_xp(self, inputs, xp):
 ]
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
+        'in_dtypes,out_dtype': _in_out_dtypes_math_functions,
+        'input': [-2, 2],
+        'contiguous': [None, 'C'],
+        'alpha_range': [(-2.0, 0.0), 0.0, (0.0, 2.0), Unspecified],
+    })
+    # Special values
+    + chainer.testing.product({
+        'shape': [(2, 3)],
+        'in_dtypes,out_dtype': _in_out_float_dtypes_math_functions,
+        'input': [0, float('inf'), -float('inf'), float('nan')],
+        'skip_backward_test': [True],
+        'skip_double_backward_test': [True],
+        'alpha_range': [(-2.0, 0.0), 0.0, (0.0, 2.0), Unspecified],
+    })
+))
+class TestElu(UnaryMathTestBase, op_utils.NumpyOpTest):
+
+    def setup(self):
+        if isinstance(self.alpha_range, tuple):
+            l, u = self.alpha_range
+            self.alpha = random.uniform(l, u)
+        elif self.alpha_range in (None, Unspecified):
```

Comment:
Is `Unspecified` necessary?
With 
`'alpha_range': [(-2.0, 0.0), (0.0, 2.0),  0.0, 1.0]`

We will have
```Python
if isinstance(self.alpha_range, tuple):
  l, u = self.alpha_range
  self.alpha = random.uniform(l, u)
else:
  self.alpha = self.alpha_range
```
Which will have the same behavior as it does now.